File: app/src/main/java/com/meghdut/upsilent/network/CreditResponse.java
package com.meghdut.upsilent.network;

import com.meghdut.upsilent.models.Cast;
import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;

/**
 * Created by Meghdut Mandal on 10/02/17.
 */

public class CreditResponse {

    @SerializedName("cast")
    private ArrayList<Cast> cast;

    public ArrayList<Cast> getCast() {
        return cast;
    }

    public void setMovieCast(ArrayList<Cast> cast) {
        this.cast = cast;
    }

}

File: app/src/main/java/com/meghdut/upsilent/network/CreditResponse.kt
package com.meghdut.upsilent.network

import com.google.gson.annotations.SerializedName
import com.meghdut.upsilent.models.Cast
import java.util.*

/**
 * Created by Meghdut Mandal on 10/02/17.
 */
class CreditResponse {
    @SerializedName("cast")
    var cast: ArrayList<Cast> = arrayListOf()
        private set

    fun setMovieCast(cast: ArrayList<Cast>) {
        this.cast = cast
    }
}